doc/user_guide/model.rst

```diff
@@ -716,9 +746,9 @@ by hand.
     :py:meth:`~.model.Model.enable_adjust_position` and
     :py:meth:`~.model.Model.disable_adjust_position`
 
-Also, :py:meth:`~.model.Model.enable_adjust_position` provides an interactive
+Also, :py:meth:`~.models.model1D.Model1D.enable_adjust_position` provides an interactive
 way of setting the position of the components with a well define position.
-:py:meth:`~.model.Model.disable_adjust_position` disables the tool.
+:py:meth:`~.models.model1D.Model1D.disable_adjust_position` disables the tool.
```

Comment:
Agree. The tilde removes everything but the last bit from what the users see anyway. The first bits are only important for where the link in the doc leads.